addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -118,11 +118,6 @@ def _get_uom_unece_code(self, line):
             return UOM_TO_UNECE_CODE.get(xmlid[line.product_uom_id.id], 'C62')
         return 'C62'
 
-    def _find_value(self, xpath, tree):
-        # avoid 'TypeError: empty namespace prefix is not supported in XPath'
-        nsmap = {k: v for k, v in tree.nsmap.items() if k is not None}
-        return find_xml_value(xpath, tree, nsmap)
```

Comment:
What about:
```suggestion
    def _find_value(self, xpath, tree, nsmap=False):
        # avoid 'TypeError: empty namespace prefix is not supported in XPath'
        nsmap = nsmap or {k: v for k, v in tree.nsmap.items() if k is not None}
        return find_xml_value(xpath, tree, nsmap)
```
and then in the ubl_20 file:

```py
def _find_value(self, xpath, tree):
    return super()._find_value(xpath, tree, UBL_NSMAP)
```
And then, the diff is like ~10 lines